server/test/parameters/CoreConfigTest.java

```diff
@@ -16,14 +16,11 @@
  *
  */
 
-package com.vaticle.typedb.core.server.test;
+package com.vaticle.typedb.core.server.parameters;
```

Comment:
we move the `Config` testing into a mirror package (eg. `//server/test` to `//server/test/parameters`) so that we can tighten some access methods to be `package-private` instead of `public`.